chapter_3/CMakeLists.txt

```diff
@@ -11,7 +11,9 @@ include_directories("${PROJECT_SOURCE_DIR}/libflandmark")
 FILE(COPY "haarcascade_frontalface_alt.xml" DESTINATION ${CMAKE_BINARY_DIR})
 FILE(COPY "flandmark_model.dat" DESTINATION ${CMAKE_BINARY_DIR})
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++")
+set(CMAKE_C_COMPILER "/usr/bin/clang")
+set(CMAKE_CXX_COMPILER "/usr/bin/clang++")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -stdlib=libc++ -stdlib=libstdc++")
```

Comment:
Due to the directive "-stdlib=libc++" - it's a Clang option and tells to use LLVM libc++ standard library.

See here: [http://stackoverflow.com/questions/24419832/unrecognized-command-line-option-stdlib-libc-with-macports-gcc48](url)